### DISCUSSION TITLE
Created a stanging site, feed now not working on the main domain

### DISCUSSION THREAD

**Discussion by ghost** [code4recovery/12-step-meeting-list#731] [Q&A] [answered]
created a staging site until it was developed to launch to the public.  We launched last week but now the feed still contains the stanging url insted of the main url.  any suggestions?  thank you!

**Comment by tech2serve** [comment]
Start with refreshing permalinks: `Settings-->Permalinks`. The scroll down and click Save Changes.

When you say "feed", I want to confirm you mean the feed that should be providing the JSON list of meetings to Meeting Guide, and not simply your meetings page?

If that doesn't resolve things, please post your website URL and and Feed link (set your sharing to Open). Then I can inspect things better.

**Comment by ghost** [comment]
Hi thanks for the reply.

https://area63aa.org

Public Feed    https://area63aa.org/wp-admin/admin-ajax.php?action=meetings


Is there anything else you would need to look into this?

I truly appreciate your help!!!!

**Comment by ghost** [comment]
I think the meeting details issue is corrected from your permalink refresh
suggestion.  The other issue we are having is the city name is not listed
on the meeting list page.  Any suggestions? See attached image screenshot.
Thank you so much!!

[image: image.png]

**Comment by brianw-area24** [comment, marked as answer]
I'm noticing a couple things when I look at the "Public Feed" URL you provided, and when I import this feed into my test site:

- Most (all) the meetings have both Regions and Sub Regions. The Regions appear to have the city/town, and the Sub Regions have "SD", "SD 57105", etc
- I'm seeing more meetings than I'd expect outside of South Dakota. For example https://area63aa.org/meetings/https-us04web-zoom-us-j-897492804/, this meeting looks like it should be in Sturgis, but the address is San Jose, CA
- Some of the other information doesn't appear to be in the correct fields

I'm not seeing the image you tried to attach. That might be helpful. Can you try again?

I'm also curious, how did you get the meetings into this site? Did they come from CSV file(s)? Feed(s) on the Imports & Settings page? Other?

**Reply by MTecknology** [reply]
For what it's worth, /this/ seems like the correct answer. After getting involved and looking at source data, it looks very much like some quotes were stripped prior to an upload and then the TSML plugin tried to guess a location based on the address fragment.
